test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_ontimeinterval.test

```diff
@@ -439,7 +439,7 @@ Content-Type: application/xml
 </notifyContextRequest>
 =======================================
 NUMBER: 16
-TIMES: REGEX((0, 2, 4, 5, 9, 13, 14, 24, 34, 35, 37, 39, 53, 65, 67, 69|0, 2, 4, 5, 9, 13, 14, 24, 34, 35, 37, 39, 51, 63, 67, 69))
+TIMES: REGEX((0|1), (1|2|3), (3|4), (5|6), (9|10), (13|14), (14|15|16), (24|25|26), (34|35|36), (35|36|37|38), (37|38|39|40), (39|40|41|42), (51|52|53|54), (63|65|66), (67|72), (69|74))
```

Comment:
Not sure if it is a good idea using "blur" times this way (I have seen it also in other .test), as it makes the test harness weaker.

Maybe an way of specifying two alternative block could be better, something in this way:

```
=======================================
 NUMBER: 16
#if valgrrind
TIMES: REGEX((0|1), (1|2|3), (3|4), (5|6), (9|10), (13|14), (14|15|16), (24|25|26), (34|35|36), (35|36|37|38), (37|38|39|40), (39|40|41|42), (51|52|53|54), (63|65|66), (67|72), (69|74))
#else
TIMES: REGEX((0, 2, 4, 5, 9, 13, 14, 24, 34, 35, 37, 39, 53, 65, 67, 69|0, 2, 4, 5, 9, 13, 14, 24, 34, 35, 37, 39, 51, 63, 67, 69))
#endif
...
```

However, probably that is very hard for this PR. Maybe we can create an issue to work on it in the future.